pkg/langruntime/langruntimetestutils.go

```diff
@@ -4,117 +4,118 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/sirupsen/logrus"
-	yaml "gopkg.in/yaml.v2"
 	"k8s.io/api/core/v1"
 	"k8s.io/client-go/kubernetes/fake"
 )
 
 // AddFakeConfig initializes configmap for unit tests with fake configuration.
 func AddFakeConfig(clientset *fake.Clientset) {
 
-	var runtimeImages = []RuntimeInfo{{
-		ID:             "python",
-		DepName:        "requirements.txt",
-		FileNameSuffix: ".py",
-		Versions: []RuntimeVersion{
-			{
-				Name:         "python27",
-				Version:      "2.7",
-				InitImage:    "tuna/python-pillow:2.7.11-alpine",
-				RuntimeImage: "kubeless/python@sha256:0f3b64b654df5326198e481cd26e73ecccd905aae60810fc9baea4dcbb61f697",
-
-				ImagePullSecrets: []ImageSecret{
-					{ImageSecret: "p1"}, {ImageSecret: "p2"},
-				},
-			}, {
-				Name:    "python34",
-				Version: "3.4",
-				ImagePullSecrets: []ImageSecret{
-					{ImageSecret: "p1"}, {ImageSecret: "p2"},
+	runtimeImages := `[
```

Comment:
just thinking that this example configuration seems unnecessary complex, why did you need to modify it?

Reply:
The problem here was previously we were using structs to initialize the runtime-images. While doing the same with liveness Probe it did not work. Since v1.Probe has an embeded struct `Handler`. This is always set to null when we are unmarshalling it. Only way it works is passing the configuration as string. This is the reason I reworked the configuration to be a json string.

Reply:
Got it. Can you try to make it something simple like a single runtime with only one version? I don't think all the info here is being used.

Reply:
Done.